### PR TITLE
fix: #55946 Added comment snippet variable

### PR DESCRIPTION
**Pull request by karanisverma**
Added comment snippet variable
it fixes following issue 
https://github.com/Microsoft/vscode/issues/55946

### CODE REVIEW

File: src/vs/editor/contrib/snippet/snippetVariables.ts [hunk 1]
```diff
@@ -180,7 +184,26 @@ export class ClipboardBasedVariableResolver implements VariableResolver {
...
+	resolve(variable: Variable): string | undefined {
+		const { name } = variable;
+		const language = this._model.getLanguageIdentifier();
+		const comments = LanguageConfigurationRegistry.getComments(language.id);
+		if (name === 'LINE_COMMENT') {
+			return comments.lineCommentToken;
+		} else if (name === 'BLOCK_COMMENT_START') {
+			return comments.blockCommentStartToken;
+		} else if (name === 'BLOCK_COMMENT_END') {
+			return comments.blockCommentEndToken;
```
[the remark this answers is not in the document]
Comment (karanisverma) [reply src/vs/editor/contrib/snippet/snippetVariables.ts]: Thanks, I have made changes as per your suggestion.

**Comment by jrieken** [comment]
@karanisverma The build has failed due to a strict null error. You can them locally via `F1 > Run Task > Strict Null Checks`

**Comment by karanisverma** [comment]
Thanks for the info, Looking into it :)

**Comment by karanisverma** [comment]
@jrieken I have fixed the issue and I have checked locally by following the steps you mentioned above. 
Sharing log for test below:

```
> Executing task: npm run strict-null-check <


> code-oss-dev@1.30.0 strict-null-check /Users/karanverma/me/oss/vscode
> tsc -p src/tsconfig.strictNullChecks.json


Terminal will be reused by tasks, press any key to close it.
``` 

build process seems to fail in following file 
`src/vs/base/browser/ui/tree/abstractTree.ts(205,12): error TS2532: Object is possibly 'undefined'.`

I guess it's not because of the changes in this PR. Please let me know if there is anything I have to fix at my end.

**Comment by karanisverma** [comment]
@jrieken I have pulled code from master, tests are passing now 🙌🏼

**Review by jrieken — APPROVED** [review]
lgtm

**Comment by karanisverma** [comment]
Anytime :), Thanks you for guiding me 🙂

**Pull request merged**